doc/user/content/get-started/quickstart.md

```diff
@@ -447,66 +436,21 @@ Let's see this in action.
     (2 rows)
     ```
 
-1. Once the `r2` replica is ready (`ready=t`), drop the `r1` replica to simulate a failure:
-
-    ```sql
-    DROP CLUSTER REPLICA compute_qck.r1;
-    ```
-
-1. Query the indexed view that you created in `compute_qck` earlier:
-
-    ```sql
-    SELECT * FROM avg_bids;
-    ```
-    <p></p>
-
-    ```nofmt
-            item        |    average_bid
-    --------------------+--------------------
-     Custom Art         | 49.770776201263864
-     Gift Basket        |   49.8909070204407
-     City Bar Crawl     | 50.056635368698046
-     Best Pizza in Town |  50.50023551577956
-     Signed Memorabilia |  50.11854192264935
-    (5 rows)
-
-    Time: 23.537 ms
-    ```
-
-    As you can see, the results are available despite the failure of one of the cluster's replicas.
+    When the new replica shows `ready=t`, it has caught up with the original replica. From that moment forward, they are logical clones that do the same computation and hold the same results.
```

Comment:
```suggestion
    When the new replica shows `ready=t`, Materialize has acquired physical compute resources for the new replica. From that moment forward, they are logical clones that do the same computation. It may take some time for the new replica to catch up to the old replica.
```

Reply:
LGTM.